src/commands/create.js

```diff
@@ -89,6 +90,16 @@ export default asyncCommand({
 			info(`Assuming you meant ${repo}...`);
 		}
 
+		// Use `--name` value or `dest` dir's name
+		argv.name = argv.name || argv.dest;
+
+		const npmPackageNameErrors = validateNpmPackageName(argv.name).errors;
+
+		if (npmPackageNameErrors) {
+			const npmPackageNameMsg = npmPackageNameErrors.join(', ');
+			return error(`${argv.name} is not a valid package name. ${npmPackageNameMsg}`, 1);
```

Comment:
can you add a `chalk` message before this?

Reply:
Not sure what the chalk message would have in it. Doesn't the `error` method from the utils already display the message given?

Reply:
You are right. Internally `error` method in `utils` uses chalk itself. 

@prateekbh that is totally valid. 😄

Reply:
gosh I need to do my revision for this codebase